handbook.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: contribute
+title: Handbook
+---
+
+# How we make things
+
+## Branching and SCM
+
+We use git for source control. Our projects in
+[theforeman](https://github.com/theforeman) follow this model for branching:
+
+* **develop**
+    * All pull requests are merged here.
+    * Contains nightlies code. It's relatively stable, but we make no promises about it.
+    * [`test_develop` in Jenkins](http://ci.theforeman.org/job/test_develop/) ensures our tests are passing at all times for various configurations.
+* **release-stable (i.e. 1.10-stable)** -
+    * Code is cherry-picked using `git cherry-pick -m` into this branch.
+    * It contains the latest stable code for a release cycle, 1.8-stable contains 1.8.1, 1.8.2, 1.8.3...
+    * Typically, we will announce the start of a new branch
+
+And we tag releases:
+
+* **release (i.e. 1.10)**
+    * Code is cherry-picked using `git cherry-pick -m` into this tag.
+    * It contains the latest stable code for a *specific release*, for instance, 1.10.0.
+
+We do not force push to any of these branches. If a commit must be reverted, use `git-revert`, and push the reverted commit.
+
+Foreman projects strive to have a linear history ([like this](http://i.imgur.com/RMFS3bt.png)). A way to achieve this is to never create merge commits, by always rebasing your changes with develop `git pull --rebase upstream develop`. This allows us to easily revert our code to any point in time, which is helpful to detect in what commit we introduced an error using `git-bisect`.
+
+## Changelog
+
+TODO - Try to describe how this looks https://www.mozilla.org/en-US/firefox/39.0/releasenotes/
```

Comment:
The repo CHANGELOGs are maintained by scripts, while release notes are part of the docs process.. it may not be immediately relevant to this handbook (in the context of making patches to core).